src/bdscript/onlyBotChannelPerms.md

```diff
@@ -8,5 +8,5 @@ $onlyBotChannelPerms[channelID;permissions;errorMessage]
 
 ### Breakdown
 - `channelID` - The channel to check the permissions for. Use `$channelID` for the current channel.
-- `permissions` - The [permissions](https://nilpointer-software.github.io/bdfd-wiki/guides/permissions.html) that the bot needs to execute the command. Separate permissions with `;`.
+- `permissions` - The [permissions](/src/resources/permissions.md) that the bot needs to execute the command. Separate permissions with `;`.
```

Comment:
Please correct arguments in the full function
[JSON function data](https://botdesignerdiscord.com/public/api/function/$onlyBotChannelPerms[])